backend/src/api/Models/ApiDbContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿using api.Maps;
+using Microsoft.EntityFrameworkCore;
+
+namespace api.Models
+{
+    public class ApiDbContext : DbContext
+    {
+        public ApiDbContext(DbContextOptions<ApiDbContext> options) : base(options)
+        {
+
+        }
+
+        public DbSet<Blog> Blogs { get; set; }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
I believe this would be better placed in `Startup.cs` like here https://docs.microsoft.com/en-us/aspnet/core/data/ef-mvc/intro?view=aspnetcore-6.0#register-the-schoolcontext if We do not create an infrastructure project